src/openforms/js/components/admin/form_design/variables/VariablesEditor.stories.js

```diff
@@ -652,6 +669,21 @@ export const WithObjectsAPIAndTestRegistrationBackends = {
       ],
     },
   },
+  play: async ({canvasElement}) => {
+    const canvas = within(canvasElement);
+
+    const registrationTab = canvas.getByRole('tab', {name: 'Registratie'});
+    await userEvent.click(registrationTab);
+
+    // With multiple backends configured, and two of them introduce registration variables,
+    // the plugin headings should be visible.
+    await expect(canvas.getByRole('heading', {name: 'Objects API registration'})).toBeVisible();
+    await expect(canvas.getByRole('heading', {name: 'Test plugin'})).toBeVisible();
```

Comment:
same remark here about the `await expect` - likely what you need is the `await canvas.findByRole(...)` inside the expect